core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1005,9 +1005,10 @@ private[spark] object Utils extends Logging {
 
   /**
    * Return the string to tell how long has passed in milliseconds.
+   * @param startTimeNs - a timestamp in nanoseconds returned by `System.nanoTime`.
    */
-  def getUsedTimeMs(startTimeMs: Long): String = {
-    " " + (System.currentTimeMillis - startTimeMs) + " ms"
+  def getUsedTimeMs(startTimeNs: Long): String = {
```

Comment:
See above; I think the method name would have to change to end with "Ns"

Reply:
I kept the "Ms" suffix because the method outputs result string in milliseconds. I could rename it to `getUsedTimeNs` but this would look slightly inconsistent. The method name indicates "used time in nanoseconds" but actually outputs milliseconds.

Reply:
Renamed it to `getUsedTimeNs`